ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -478,7 +478,7 @@ StatusDialog {
                             if (!valid)
                                 return 0
 
-                            return parseFloat(text.replace(",", "."))
+                            return parseFloat(text.replace(LocaleUtils.userInputLocale.decimalPoint, "."))
```

Comment:
Great idea to switch from comma to locale dependent decimal point. According to https://learn.microsoft.com/en-us/globalization/locale/number-formatting there might be some other characters for decimal part separation.
Also interesting but maybe not relevant case with Switzerland, where ',' is for decimal separation, but for currencies they use '.'

Reply:
Yeah, sometimes it's mixed, some locales also use `,` as the thousands separator so we should never assume `,` might be always the decimal separator